docs/xks/operator-guide/architecture-and-design.md

```diff
@@ -0,0 +1,117 @@
+---
+id: architecture-and-design
+title: Architecture and design
+---
+
+import useBaseUrl from '@docusaurus/useBaseUrl';
+
+## Overview
+
+[Xenit Kubernetes Framework](https://github.com/search?q=topic%3Axenit-kubernetes-framework+org%3AXenitAB+fork%3Atrue) (XKF) is the open source building blocks for a service Xenit AB provides customers: [Xenit Kubernetes Service](https://xenit.se/it-tjanster/kubernetes-eng/) (XKS)
```

Comment:
"... *are* the open source building blocks ..."

Reply:
done!